src/components/Personalization/actions/setText.js

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { showElements } from "../flicker";
+
+export default collect => {
+  return (settings, event) => {
+    const { elements, prehidingSelector } = event;
+    const { content, meta } = settings;
+
+    elements.forEach(element => {
+      element.textContent = content;
+    });
+
+    // after rendering we should show remove the flicker control styles
```

Comment:
Typo in "show remove"? It looks like it's in `setHtml.js` as well.

Reply:
Done.